delfin/utils.py

```diff
@@ -596,6 +624,20 @@ def write_remote_file(ssh, filename, contents, as_root=False):
     ssh.exec_command(cmd2)
 
 
+def change_capacity_to_bytes(unit):
+    if unit == 'TB':
+        result = 1024 * 1024 * 1024 * 1024
+    elif unit == 'GB':
+        result = 1024 * 1024 * 1024
+    elif unit == 'MB':
+        result = 1024 * 1024
+    elif unit == 'KB':
+        result = 1024
+    else:
```

Comment:
This `else` is not safe. If someone pass unit like 'Kb' or 'kb', it would come to this `else` branch and return 1.

Reply:
I change the str upper first now